openedx_email_extensions/templates/email_layout/mc_base.html

```diff
@@ -134,20 +135,23 @@
                                 extra_footer_template = proxy_settings.EXTRA_FOOTER_TEMPLATE
                               %>
                               % if extra_footer_template:
+
                                 <%include file="${ template_path_finder(extra_footer_template) }" />
-                              % endif
```

Comment:
removing this if has the side effect of making the `extra_footer_template` the same as the footer_override. I think we should not change the signature of the microsite variables for this.